### DISCUSSION TITLE
Why highres_fix_scale does not work?

### DISCUSSION THREAD

**Discussion by Felix0231** [kohya-ss/sd-scripts#743] [Q&A] [answered]
I use the command **`--highres_fix_scale 0.5 --highres_fix_steps 10 --strength 0.5`** to generate detailed imgs, the initial size is 512*768 ,but I found the output size is the same. Is anyone meet the same problem? I want to kown whether it is the code bug.

**Comment by kohya-ss** [comment, marked as answer]
In high_res_fix, please specify the final resolution to `--W` and `--H` options, like `--W 1024 --H 1536`  in your case.

**Reply by Felix0231** [reply]
It works!!! Thanks!!!
